js/schedule.js

```diff
@@ -6,17 +6,17 @@ function Schedule(options) {
         // when you create a Schedule() instance on the page
         schedule.sourceJSON = 'sessions.json';
         schedule.$container = $('#schedule');
-        schedule.$toggles = $('<ul>').appendTo('#schedule-controls').hide();
+        schedule.$toggles = $('<ul>').appendTo('#schedule-controls');
         // if true, avoids using history.back(), which doesn't work offline
         schedule.offlineMode = false;
 
         // TODO: determine list of unique tab names and dates
         // after loadSessions() gets actual session data
         schedule.tabList = [
-            { name: 'Friday', tabDate: new Date(2014,9,26) },
-            { name: 'Saturday', tabDate: new Date(2014,9,25) },
-            { name: 'Sunday', tabDate: new Date(2014,9,26) },
-            { name: 'All' }
+            { name: 'Friday', displayName: 'Fri', tabDate: new Date(2015,11,6) },
+            { name: 'Saturday', displayName: 'Sat', tabDate: new Date(2015,11,7) },
+            { name: 'Sunday', displayName: 'Sun', tabDate: new Date(2015,11,8) },
```

Comment:
Pretty sure these all need to be (2015, 10, X), because javascript zero-bases the month value. (This bites me _every time_)